homeassistant/components/deconz/climate.py

```diff
@@ -97,28 +97,28 @@ async def async_setup_entry(
     gateway.entities[DOMAIN] = set()
 
     @callback
-    def async_add_climate(_: EventType, climate_id: str) -> None:
-        """Add climate from deCONZ."""
-        climate = gateway.api.sensors.thermostat[climate_id]
-        if not gateway.option_allow_clip_sensor and climate.type.startswith("CLIP"):
+    def async_add_sensor(_: EventType, sensor_id: str) -> None:
```

Comment:
This change is confusing? Renaming the climate stuff to sensor?

Reply:
Fixed